### DISCUSSION TITLE
The problem of copying a link to a page

### DISCUSSION THREAD

**Discussion by adminuss** [docmost/docmost#321] [Help / Questions]
When you try to copy the link to the wanderer, a message appears that the link has been copied, but when you paste the copied link,
![Screenshot 2024-09-07 005122](https://github.com/user-attachments/assets/386ca2b3-5a6f-4930-b341-4a9958a72a36)
 the link is inserted, and the text from the clipboard is inserted.

![image](https://github.com/user-attachments/assets/3615e7ea-4520-4f36-bb2f-1f1f071a8d18)

**Comment by Philipinho** [comment]
Did you have HTTPS setup?

**Reply by Hypernova007** [reply]
I have the same problem. I don't have HTTPS.

**Reply by Hypernova007** [reply]
Using HTTPS solves the problem.
